test_common/harness/imageHelpers.h

```diff
@@ -631,7 +631,14 @@ class CoordWalker
 
 extern int  DetectFloatToHalfRoundingMode( cl_command_queue );  // Returns CL_SUCCESS on success
 
-int inline is_half_nan( cl_ushort half ){ return (half & 0x7fff) > 0x7c00; }
+// sign bit: don't care, exponent: maximum value, significand: non-zero
+static int inline is_half_nan( cl_ushort half ){ return ( half & 0x7fff ) > 0x7c00; }
+
+// sign bit: don't care, exponent: zero, significand: non-zero
+static int inline is_half_denorm( cl_ushort half ){ return IsHalfSubnormal( half ); }
+
+// sign bit: don't care, exponent: zero, significand: zero
+static int inline is_half_zero( cl_ushort half ){ return ( half & 0x7fff ) == 0; }
```

Comment:
Please use `cl_half` instead of `cl_short`. These functions are also good candidates to move to the headers.

Reply:
Added to #511